drivers/lora/sx1276.c

```diff
@@ -388,8 +400,10 @@ static int sx1276_lora_recv(struct device *dev, u8_t *data, u8_t size,
 	 * As per the API requirement, timeout value can be in ms/K_FOREVER/
 	 * K_NO_WAIT. So, let's handle all cases.
 	 */
-	ret = k_sem_take(&dev_data.data_sem, timeout == K_FOREVER ? K_FOREVER :
-			 timeout == K_NO_WAIT ? K_NO_WAIT : K_MSEC(timeout));
+	ret = k_sem_take(&dev_data.data_sem,
+			 K_TIMEOUT_EQ(timeout, K_FOREVER) ? K_FOREVER :
```

Comment:
i don't think you need that anymore. `k_sem_take(..., timeout)` should be enough.

Reply:
Ack